distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -153,6 +153,7 @@ ALERT;ElectroluxAir Binding: The binding has been removed since the Electrolux D
 ALERT;JavaScript Automation: The isJsInstanceOfJavaType method of the utils namespace has been removed. Use JavaScript's instanceof operator instead.
 ALERT;MQTT Binding: Newly discovered Home Assistant things will use simplified thing type and channel IDs. Delete and re-create your things to opt in to the new style. In subsequent a openHAB release, existing things will also convert to the new style.
 ALERT;Pentair Binding: EasyTouch thing has been renamed to more generic Controller and all channels have been organized into groups. You will need to reconfigure your setup to the new thing structure.
+ALERT;MeteoAlert Binding: The underlaying API stopped delivering data in May 2023. Binding has been removed and is now replaced by Météo France Binding based on a new API.
```

Comment:
```suggestion
ALERT;MeteoAlerte Binding: The underlying API stopped delivering data in May 2023. Binding has been removed and is now replaced by Météo France Binding based on a new API.
ALERT;MQTT Binding: Newly discovered Home Assistant things will use simplified thing type and channel IDs. Delete and re-create your things to opt in to the new style. In subsequent a openHAB release, existing things will also convert to the new style.
ALERT;Pentair Binding: EasyTouch thing has been renamed to more generic Controller and all channels have been organized into groups. You will need to reconfigure your setup to the new thing structure.
```

Reply:
@ccutrer If you find some time today to fix this, it can go into the M2 milestone build. Thanks!